storybook/webpack.config.js

```diff
@@ -23,15 +24,9 @@ module.exports = {
             options: {
               includePaths: [
                 path.resolve(__dirname, '../sass/patternfly-react'),
-                path.resolve(__dirname, '../node_modules/patternfly/dist/sass'),
-                path.resolve(
-                  __dirname,
-                  '../node_modules/patternfly/node_modules/bootstrap-sass/assets/stylesheets'
-                ),
-                path.resolve(
-                  __dirname,
-                  '../node_modules/patternfly/node_modules/font-awesome-sass/assets/stylesheets'
-                )
+                path.resolve(__dirname, `../${pkg.sassIncludes.patternfly}`),
```

Comment:
I really like this approach, but this can also be simplified to
```javascript
Object.keys(pkg.sassIncludes)
  .map(oneInclude => path.resolve(__dirname, `../${pkg.sassIncludes[oneInclude]}`))
```

Reply:
Thanks Karel, I agree - will do the code change as suggested.

Reply:
```diff
                 path.resolve(__dirname, '../sass/patternfly-react'),
-                path.resolve(__dirname, `../${pkg.sassIncludes.patternfly}`),
-                path.resolve(__dirname, `../${pkg.sassIncludes.bootstrap}`),
-                path.resolve(__dirname, `../${pkg.sassIncludes.fontAwesome}`)
+                ...Object.values(pkg.sassIncludes).map(includePath =>
+                  path.resolve(__dirname, `../${includePath}`)
+                )
```